fe/model.py

```diff
@@ -91,19 +93,34 @@ def predict(self, ff_params: list, mol_a: Chem.Mol, mol_b: Chem.Mol, core: np.nd
 
             unbound_potentials, sys_params, masses, coords = rfe.prepare_host_edge(ff_params, host_system, min_host_coords)
 
+            harmonic_bond_potential = unbound_potentials[0]
+            group_idxs = get_group_indices(get_bond_list(harmonic_bond_potential))
+
             x0 = coords
             v0 = np.zeros_like(coords)
 
             seed = 0
 
+            temperature = 300.0
+            pressure = 1.0
+
             integrator = LangevinIntegrator(
-                300.0,
+                temperature,
                 1.5e-3,
                 1.0,
                 masses,
                 seed
             )
 
+            barostat = MonteCarloBarostat(
```

Comment:
I need to rethink the API for the barostat needs to be rethought, right now it feels clumsy

Reply:
Not that bad imo